python.d/python_modules/bases/FrameworkServices/UrlService.py

```diff
@@ -75,20 +75,30 @@ def _get_raw_data(self, url=None, manager=None):
         :return: str
         """
         try:
-            url = url or self.url
-            manager = manager or self._manager
-            response = manager.request(method='GET',
-                                       url=url,
-                                       timeout=self.request_timeout,
-                                       retries=1,
-                                       headers=manager.headers)
+            status, data = self._get_raw_data_with_status(url, manager)
+            if status == 200:
```

Comment:
please move the code after  
```python
status, data = self._get_raw_data_with_status(url, manager)
```
to the else block of `try/except` or just after `except`

Reply:
IMO, that makes code readability harder, as it splits successful branch execution somewhere else. But have it your way